srcpkgs/Adapta/template

```diff
@@ -3,12 +3,12 @@ pkgname=Adapta
 version=3.95.0.11
 revision=1
 noarch=yes
-build_style=gnu-configure
 wrksrc="adapta-gtk-theme-$version"
-hostmakedepends="automake glib-devel gnome-shell inkscape parallel pkg-config
+build_style=gnu-configure
+hostmakedepends="automake glib-devel inkscape parallel pkg-config
  procps-ng sassc"
 makedepends="glib-devel librsvg-devel"
-short_desc="An adaptive Gtk+ theme based on Material Design Guidelines"
+short_desc="Adaptive Gtk+ theme based on Material Design Guidelines"
```

Comment:
short_desc is used changing, if you modify it then you need to revbump=

if you are revbumping also fix the license.

Reply:
Gotcha. What's up with the license?